src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -105,52 +94,21 @@ class Breakpoints extends Component<Props> {
   }
 
   renderBreakpoint(breakpoint) {
-    const locationId = breakpoint.locationId;
-    const line = breakpoint.location.line;
-    const column = breakpoint.location.column;
-    const isCurrentlyPaused = breakpoint.isCurrentlyPaused;
-    const isDisabled = breakpoint.disabled;
-    const isConditional = !!breakpoint.condition;
-    const isHidden = breakpoint.hidden;
-
-    if (isHidden) {
+    if (!breakpoint.id) {
```

Comment:
So I'm not a fan of this, it's a bandage, but React is throwing key errors every time I add a breakpoint because, upon first render, breakpoints have no `id` (undefined), and I'm not able to add a representative key